media-gfx/solvespace/solvespace-2.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DXFRW_COMMIT="87ff1082f49b1286a033ac8f38fe8dde0975bb8b"
+DXFRW_PV="0.6.3"
+DXFRW_PN="libdxfrw"
+DXFRW_P="${DXFRW_PN}-${DXFRW_PV}-${DXFRW_COMMIT}"
+
+inherit cmake-utils
+
+DESCRIPTION="Parametric 2d/3d CAD"
+HOMEPAGE="http://solvespace.com"
+SRC_URI="https://github.com/solvespace/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/solvespace/${DXFRW_PN}/archive/${DXFRW_COMMIT}.zip -> ${DXFRW_P}.zip"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-cpp/gtkmm:2.4
+	dev-cpp/pangomm:1.4
+	dev-libs/json-c:=
+	media-libs/fontconfig
+	media-libs/freetype:2
+	media-libs/glew:0=
+	media-libs/libpng:0=
+	virtual/opengl"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+GIT_COMMIT_HASH="4d1e1341d926ac356b295d5cc3d61c7a6cd7d07d"
+
+src_prepare() {
+	default
+	rm -r "extlib/${DXFRW_PN}" || die  "rm extlib/${DXFRW_PN} failed"
+	mv -T "${WORKDIR}/${DXFRW_PN}-${DXFRW_COMMIT}" "extlib/${DXFRW_PN}" || die "move ${DXFRW_PN}-${DXFRW_COMMIT} failed"
+	# fixed in master already
+	sed -i '/include(GetGitCommitHash)/d' CMakeLists.txt || die 'remove GetGitCommitHash by sed failed'
+	sed -i 's/CHAR_WIDTH/CHAR_WIDTH_/' src/textwin.cpp || die 'sed failed'
```

Comment:
You can pass multiple files to `sed`. Also use `sed -i -e '...' files` for consistency.